benches/serialization/bench_serialization.rs

```diff
@@ -0,0 +1,114 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate protobuf;
+
+use test::Bencher;
+use protobuf::Message;
+use kvproto::eraftpb::Entry;
+use kvproto::raft_cmdpb::{RaftCmdRequest, Request, CmdType};
+use std::time::{Instant, Duration};
+use std::string::String;
+
+#[inline]
+fn encode_one_cmd() -> Vec<u8> {
```

Comment:
I think we should prepare the request for the benchmark, creating request every time may affect the bench.